### ISSUE TITLE
Events: Relevance improvements

### ISSUE THREAD

**Issue by agnesgaroux** [wellcomecollection/content-api#123]
Apply changes to relevance sorting https://github.com/wellcomecollection/docs/tree/main/rfcs/061-content-api-next-steps#sorting

> We suggest adding these field to calculate the relevance score:
> format
> audiences
> series

Might need consultation for what weight to assign to each.

**Comment by rcantin-w** [comment]
Moving to blocked as not a priority and has taken up too much time already, will back up when I'm done with the work that was planned.

**Comment by rcantin-w** [comment]
Unblocked, new pipeline was created and we have access to new results. 
Will review them with Lauren tomorrow; if agreed upon, will move towards making new pipeline the live one.

**Comment by rcantin-w** [comment]
Changes are now live
